ci-operator/config/openstack-k8s-operators/openstack-baremetal-operator/openstack-k8s-operators-openstack-baremetal-operator-master.yaml

```diff
@@ -0,0 +1,30 @@
+build_root:
+  from_repository: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make gotest
+  container:
+    from: src
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    mkdir -p ../operator/ && cp -r . ../operator
+    cd ../operator
+    pre-commit run -a --show-diff-on-failure
+  container:
+    from: openstack-operator-ci-pre-commit
```

Comment:
you also need to provide this container to this configuration file, like in [1]

[1] https://github.com/openshift/release/blob/d0dcd995c2066745c4f4778225c6a14ea483b660/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L29-L33

Reply:
Then ci-operator will build:
root -> src -> penstack-operator-ci-pre-commit

Reply:
thanks for comments I will update for the latest..

one general question @viroel what is preferred... should i force push like i did yesterday, or stack commits ontop of this one?

I see that folks are stacking commits (like eg https://github.com/openshift/release/pull/36553/commits) for corrections but it makes more sense to me to squash them into one (with a force push). Is there a preferred way?

Reply:
I don't think that there is a preferred way. I prefere to squash/amend and force push when I am fixing errors in the code. When I am just adding something, sometimes I push a new commit (since it adds a new information in commit msg)

Reply:
thank you